### ISSUE TITLE
Numpy Error

### ISSUE THREAD

**Issue by ItsSusanBoyle** [yawetag/fakesports_rss#11]
Got this error when running main.py script:

> numpy: Error importing numpy: you should not try to import numpy from
        its source directory; please exit the numpy source tree, and relaunch
        your python interpreter from there.

I'm not an avid python user so numpy isn't something I'm familiar with. How do I fix?

**Comment by yawetag** [comment]
I really should get this to send me issues that come up.

Have you installed numpy directly? Something like `pip install numpy`.

**Comment by ItsSusanBoyle** [comment]
Yeah that's already installed. Ran that again but "requirement already satisfied".
